### DISCUSSION TITLE
A bit more explanation about some properties.

### DISCUSSION THREAD

**Discussion by ScreamZ** [VirtusLab/strapi-plugin-navigation#325] [Q&A] [answered]
Hi,

I'm trying to use this plugin which I found really nice! But IMO it lacks some sort of details on specific properties, are some questions:

- What does it means « Attached to menu », what does this really imply on the backend side, what is that supposed to be used for?
- What does « Wrapper element » mean compared to « External Links »?
> Looks like it's related to elements that are just supposed to create a nested path slug.
- Not sure about what an audience is supposed to be ? What differs from custom fields?

Thanks for your time,
Andréas

**Comment by cyp3rius** [comment, marked as answer]
Hello @ScreamZ ,

- `Attached to menu` is the property which some developers using to render menu on their end differently than the whole routing based on the navigation structure. Simply not all elements are visible and added to menu itself, but routing contains them. Useful.
- It's no and external or internal url - simply the "group" of urls like dropdown menu without any redirection
- Audience is the key - value used for some kind of personalisation by some developers, based on who is rendering the navigation. It's optional and you don't need it.
